src/Model/Serializer/Offer/OfferDenormalizer.php

```diff
@@ -192,10 +192,10 @@ protected function denormalizeOffer(array $data): ImmutableOffer
 
     protected function denormalizeDescription(array $data, ImmutableOffer $offer): ImmutableOffer
     {
-        if (isset($data['description'])) {
+        if (isset($data['description']) && !empty(array_filter($data['description']))) {
```

Comment:
```suggestion
        if (!isset($data['description'])) {
            return $offer;
        }
        
        $filteredDescription = array_filter($data['description']));        
        
        if (empty($filteredDescription) {
            return $offer;
        }
        
        /* @var TranslatedDescription $description */
        $description = $this->descriptionDenormalizer->denormalize(
            $filteredDescription,
            TranslatedDescription::class,
            ...
```

Reply:
To not filter the array 2 times. But the current code is more compact. So both are fine 😉